src/policy/fees.cpp

```diff
@@ -139,22 +137,18 @@ class TxConfirmStats
 TxConfirmStats::TxConfirmStats(const std::vector<double>& defaultBuckets,
                                 const std::map<double, unsigned int>& defaultBucketMap,
                                unsigned int maxPeriods, double _decay, unsigned int _scale)
-    : buckets(defaultBuckets), bucketMap(defaultBucketMap)
+    : buckets(defaultBuckets), bucketMap(defaultBucketMap), decay(_decay), scale(_scale)
 {
-    decay = _decay;
     assert(_scale != 0 && "_scale must be non-zero");
-    scale = _scale;
     confAvg.resize(maxPeriods);
-    for (unsigned int i = 0; i < maxPeriods; i++) {
-        confAvg[i].resize(buckets.size());
-    }
     failAvg.resize(maxPeriods);
     for (unsigned int i = 0; i < maxPeriods; i++) {
+        confAvg[i].resize(buckets.size());
         failAvg[i].resize(buckets.size());
     }
 
     txCtAvg.resize(buckets.size());
-    avg.resize(buckets.size());
+    feerate_avg.resize(buckets.size());
```

Comment:
current style guide is to name prefix member variable names with m_, so this would become `m_feerate_avg`. No need to update the other member variable names - just this one because you're already touching it.

Reply:
Done